Scripts/BuildDatabank/searchDATABANK.py

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python3
+# coding: utf-8
+
```

Comment:
do we really need these shebangs?

Reply:
I would leave it just for marking a file as a standalone python executable. And it should work for both nix/macos. Do you agree?

Reply:
okay, let's keep it